src/components/Header.js

```diff
@@ -7,33 +7,28 @@ import {
   NavLink,
   Nav,
   NavbarBrand,
-  UncontrolledDropdown,
-  DropdownToggle,
-  DropdownMenu,
-  DropdownItem,
-  NavbarText,
 } from 'reactstrap';
 
-function Header(args) {
+function Header() {
   const [isOpen, setIsOpen] = React.useState(false);
 
   return (
 <div style={{
```

Comment:
Looks like the indentation is off. Please check the indentation on all your files.